program/program/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿// See https://aka.ms/new-console-template for more information
+// Exercise 1
+Console.WriteLine("Introdu primul numar");
```

Comment:
Nice! I like the way you made the code "user friendly" by explicitly have details in the output. One mention: add some new lines in the code here and there (i.e. before the comment with `//Exercise 2`) so that the code become easier to read.